### ISSUE TITLE
Disable VsTest warning

### ISSUE THREAD

**Issue by apobekiaris** [microsoft/azure-pipelines-tasks#10419]
## Note  
Issues in this repo are for tracking bugs, feature requests and questions for the tasks in this repo 

For a list:  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

If you have an issue or request for the Azure Pipelines service, use developer community instead:  

https://developercommunity.visualstudio.com/spaces/21/index.html )

## Required Information

Entering this information will route you directly to the right team and expedite traction.

**Question, Bug, or Feature?**  
*Type*: Question

**Enter Task Name**: VsTest  

list here (V# not needed):  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

## Environment
- Server - Azure Pipelines or TFS on-premises?
    
    - If using TFS on-premises, provide the version: 
    
    - If using Azure Pipelines, provide the account name, team project name, build definition name/build number: 
https://dev.azure.com/eXpandDevOps/eXpandFramework/_build/results?buildId=2070

## Issue Description

Hot to disable this warning?

> Running UI tests in parallel on the same machine can lead to errors. Consider disabling the ‘run in parallel’ option or run UI tests using a separate task.

I am aware what I am doing and my tests use separate AppDomains, I need to disable the warning as it pollutes my pipeline.

**Comment by prawalagarwal** [comment]
@apobekiaris Thanks for reporting. This warning is for users who are not aware of the impact of UI tests running in parallel. Today we do not have support for disabling warnings based on user input in vstest task. 
@PBoraMSFT can comment if we want to take this up.

**Comment by ShreyasRmsft** [comment]
@apobekiaris you can untick the UI tests option, it quite literally does nothing.
